packages/xcm-config/src/assets.ts

```diff
@@ -1,60 +1,66 @@
 import { Asset } from '@oak-foundation/xcm-types';
 
-const turing = new Asset({
+const tur = new Asset({
   key: 'tur',
   symbol: 'TUR',
+  decimals: 10,
```

Comment:
The decimals might not be global. For example, is the TUR on Turing has the same decimals on Mangata?

Reply:
The decimals of TUR on Mangata is also 10. Maybe the decimals value should be the same across chains. If the decimals is different, the asset cannot use the same symbol as "TUR".